int32 mode
string start_waypoint
string end_waypoint
string description
string start_time
